Front-end/src/Profile.jsx

```diff
@@ -98,19 +100,83 @@ const Profile = (props) => {
       })
     );
   }
+  function Followers() {
+    fetch(`http://localhost:3000/followers/${profile.id}`, {
+      method: "GET",
+      headers: {
+        "Content-Type": "Application/json",
+      },
+    }).then((data) =>
```

Comment:
Is `data` a number/int type?

Reply:
data is an object containing integers and strings so I named it data.